tests/lint-roller-markdown-api-history.spec.ts

```diff
@@ -0,0 +1,431 @@
+import { spawnSync } from 'node:child_process';
+import path, { resolve } from 'node:path';
+import { readdir, unlink, writeFile } from 'node:fs/promises';
+
+import { afterAll, beforeAll, describe, expect, it } from 'vitest';
+
+const FIXTURES_DIR = resolve(__dirname, 'fixtures');
+const MOCKUP_API_HISTORY_SCHEMA = resolve(FIXTURES_DIR, 'mockup-api-history.schema.json');
+const MOCKUP_BREAKING_CHANGES_FILE = resolve(FIXTURES_DIR, 'mockup-breaking-changes.md');
+const MOCKUP_DOCS_API_FOLDER = resolve(FIXTURES_DIR, 'api-history/mockup-api');
```

Comment:
Can we just use a `tmpdir` like in `tests/lint-roller-markdown-standard.spec.ts`? Modifying the source tree during tests is generally not a good idea, and in this case if cleanup fails to run you'd get files left over that git would want to commit. Simpler and cleaner to use a temp directory.

Reply:
Resolved in [`ed86d93` (#73)](https://github.com/electron/lint-roller/pull/73/commits/ed86d936419fdeb55f6eea8b5bf5c163ea19eafe)